tests/python/pants_test/util/test_dirutil.py

```diff
@@ -149,7 +149,7 @@ class Dir(datatype(['path'])):
   class File(datatype(['path', 'contents'])):
     @classmethod
     def empty(cls, path):
-      return cls(path, contents=b'')
+      return cls(path, contents='')
```

Comment:
Throughout the tests for comparing trees, `contents` was being encoded as byte strings in the expected output. I'm not sure why - the tests pass when unicode in Py2, and the bytes present an issue when using Py3.